config/olm/openshift/0.2.1/manifests/dynatrace-operator.v0.2.1.clusterserviceversion.yaml

```diff
@@ -17,6 +17,19 @@ metadata:
             },
             "apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
             "classicFullStack": {
+              "enabled": false,
```

Comment:
Please revert.
Will be like that for next version but for 0.2.1 it should stay with only classicFullStack enabled